packages/edit-site/src/components/page/style.scss

```diff
@@ -1,17 +1,13 @@
 .edit-site-page {
 	color: $gray-800;
 	background: $white;
-	height: calc(100% - #{$header-height});
+	height: 100%;
```

Comment:
If we do not change this to 100%, there will be space at the bottom of the page screen:

![image](https://github.com/user-attachments/assets/0469e25e-0533-442a-a204-952bade597bd)

Originally, this height was added to prevent unintended scrolling in the mobile layout.

However, now that the mobile layout matches its desktop counterpart, it's no longer necessary to subtract the header height. See https://github.com/WordPress/gutenberg/pull/64844#pullrequestreview-2265366632